internal/server/access_keys_test.go

```diff
@@ -243,53 +243,6 @@ func TestAPI_ListAccessKeys(t *testing.T) {
 		}
 	})
 
-	t.Run("delete by name", func(t *testing.T) {
```

Comment:
All of this below was moved to `TestAPI_DeleteAccessKey`. Look at the first commit to see the new test case before I extracted these cases into a separate function.

Reply:
Thanks for moving this. I had looked at doing it in the last commit and it was getting a little crazy.